custom_components/healthchecksio/__init__.py

```
@@ -58,27 +70,27 @@ async def async_setup_entry(
+    hass.data[DOMAIN_DATA][DATA_CLIENT] = HealthchecksioData(
         hass, api_key, check, self_hosted, site_root, ping_endpoint
     )
-
-    # Add binary_sensor
-    hass.async_add_job(
-        hass.config_entries.async_forward_entry_setup(
-            config_entry, Platform.BINARY_SENSOR
-        )
-    )
+    await hass.config_entries.async_forward_entry_setups(config_entry, platforms)
+    _LOGGER.debug(f"Config Entry: {config_entry.as_dict()}")
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)